Severson-Group/eMach-matlab
# Add JMAG tool (old)

This branch integrates JMAG into eMach.

This commit enables to draw some simple geometry. 
Test procedures:
1. cd to a empty directory
2. git clone https://github.com/Severson-Group/eMach.git .
3. git fetch
4. git checkout jmag_tool
5. open matlab and go to the corresponding directory
6. add all folders to path
7. open and run either one of these two files under example folder:
xfemmExampleArc.m
xfemmExampleNotchedRectangle.m
8. You see something as follows:
![image](https://user-images.githubusercontent.com/44207479/55506883-1e5d1300-561c-11e9-87d3-c47b7034d6c8.png)

I got this tested on serveron01, and it works as expected.
